content/bn/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "শিডিউলিং, প্রিএম্পশন এবং ইভিকশন"
+weight: 95
+content_type: concept
+description: >
+   কুবারনেটিসে, শিডিউলিং মানে হল নিশ্চিত করা যে পডগুলি নোডগুলির 
+   সাথে মিলিত হয়েছে কিনা যাতে কিউবলেট তাদের রান করতে পারে। 
+   প্রিএম্পশন হল স্বল্প অগ্রাধিকার পডগুলি বাতিল করার প্রক্রিয়া যাতে উচ্চ 
+   অগ্রাধিকার পডগুলি নোডগুলিতে শিডিউল করতে পারে। ইভিকশন হল 
+   রিসোর্স-ক্ষুধার্ত নোডগুলিতে এক বা একাধিক পডগুলি প্রত্যাহার করার প্রক্রিয়া।
+no_list: True
+---
+
+কুবারনেটিসে, শিডিউলিং মানে হল নিশ্চিত করা যে {{<glossary_tooltip text="পডগুলি" term_id="pod">}} 
+{{<glossary_tooltip text="নোডগুলির" term_id="node">}} সাথে মিলিত হয়েছে কিনা যাতে {{<glossary_tooltip text="কিউবলেট" term_id="kubelet">}} তাদের 
+রান করতে পারে। প্রিএম্পশন হল স্বল্প {{<glossary_tooltip text="অগ্রাধিকার" term_id="pod-priority">}} পডগুলি বাতিল করার প্রক্রিয়া
+যাতে উচ্চ অগ্রাধিকার পডগুলি নোডগুলিতে শিডিউল করতে পারে। ইভিকশন হল সম্পদ-ক্ষুধার্ত নোডগুলিতে এক বা 
```

Comment:
রিসোর্স-ক্ষুধার্ত

Reply:
![Uploading image.png…]()

Reply:
@itsPronay bhaiya is it alright now?

Reply:
looks good to me.. if there are any more possible errors then asem-hamid will let you know... @azaynul10

Reply:
alight Thank you @itsPronay bhaiya ❤️